projects/dotcms-js/src/lib/core/core-web.service.ts

```diff
@@ -261,15 +261,14 @@ export class CoreWebService {
     }
 
     private getFixedUrl(url: string): string {
-        if (url.startsWith('api')) {
+        if (url?.startsWith('api')) {
             return `/${url}`;
         }
 
-        const [version] = url.split('/');
+        const version = url ? url.split('/')[0] : '';
```

Comment:
this way because using just `?` + `[version]`, gives, https://stackoverflow.com/questions/50329128/what-is-an-intermediate-value